src/main/java/mate/academy/springbootintro/model/Book.java

```diff
@@ -38,7 +41,7 @@ public class Book {
     private String coverImage;
     private String description;
     @Column(name = "categories")
-    @ManyToMany()
+    @ManyToMany(fetch = FetchType.EAGER)

```

Comment:
```suggestion
    @ManyToMany
```
it's not a good practice, because it's bad for performance. please [read](https://stackoverflow.com/a/24154042) about these problems.